index.html

```diff
@@ -0,0 +1,459 @@
+                        </div>
+            </div>
+            </section>
+
+            
+        </article>
+    </main>
+
+    <footer>
+        <div id="footer-header">
+            <img class="logo footer-logoleft" src="assets/nytimes-logo.png" alt="New york Times Logo">
+            <a class="footer-rightlink" href="#">Go to Home Page » </a>
+        </div>
+
+        <div id="footer-links">
+            <div class="column">
+                <h4>NEWS</h4>
+                <p>Home Page</p>
+                <p>World</p>
+                <p>U.S.</p>
+                <p>Politics</p>
+                <p>Election 2020</p>
+                <p>New York</p>
+                <p>Business</p>
+                <p>Tech</p>
+                <p>Science</p>
+                <p>Sports</p>
+                <p>Obituaries</p>
+                <p>Today's Papers</p>
+                <p>Corrections</p>
+            </div>
+
+            <div class="column">
+                <h4>OPINION</h4>
+                <p>Today's Opinion</p>
+                <p>Op-Ed Columnists</p>
+                <p>Editorials</p>
+                <p>Op-Ed Contributors</p>
+                <p>Letters </p>
+                <p>Sunday Review</p>
+                <p>Video: Opinion</p>
+
+            </div>
+
+            <div class="column">
+                <h4>ARTS</h4>
+                <p>Today's Arts</p>
+                <p>Art & Design</p>
+                <p>Books</p>
+                <p>Dance</p>
+                <p>Movies</p>
+                <p>Music</p>
+                <p>Pop Culture</p>
+                <p>Television</p>
+                <p>Theater</p>
+                <p>Video: Arts</p>
+            </div>
+
+            <div class="column">
+                <h4>LIVING</h4>
+                <p>Automobiles</p>
+                <p>Crossword</p>
+                <p>Education</p>
+                <p>Food</p>
+                <p>Health</p>
+                <p>Jobs</p>
+                <p>Magagzine</p>
+                <p>Parenting</p>
+                <p>Real Estate</p>
+                <p>Style</p>
+                <p>T Magazine</p>
+                <p>Travel</p>
+                <p>Love</p>
+            </div>
+
+            <div class="column">
+                <h4>MORE</h4>
+                <p>Reader Center</p>
+                <p>Wire cutter</p>
+                <p>Live Events</p>
+                <p>The learning Network</p>
+                <p>Tool & Services</p>
+                <p>N.Y.C Events Guide</p>
+                <p>Multimedia</p>
+                <p>Photography</p>
+                <p>Videos</p>
+                <p>Newsletters</p>
+                <p>N.Y.C Stores</p>
+                <p>Time Journals</p>
+                <p>Manage my Account</p>
+
+      
+               
+            </div>
+
+            <div id="last-column" class="column">
+                <h4>SUBSCRIBE</h4>
+                <a href="#"><img src="assets/29.jpg"></a> 
```

Comment:
#### `REQUIRED`

:bulb: `img` `src` is not properly referenced, the current filename is `29.JPG` not `29.jpg`. See suggestion:
```suggestion
                <a href="#"><img src="assets/29.JPG"></a> 
```

### Do this for all other occurrences.  :wrench: